editor/src/communication/dispatcher.rs

```diff
@@ -339,10 +339,19 @@ mod test {
 		init_logger();
 		let mut editor = create_editor_with_three_layers();
 
-		let sorted_layers = editor.dispatcher.documents_message_handler.active_document().all_layers_sorted();
+		fn to_vec<'a>(paths: Vec<&'a [LayerId]>) -> Vec<Vec<LayerId>> {
```

Comment:
A bit confusing with the std `.to_vec` (copying outer vec). Perhaps `map_to_vec`?

Reply:
yeah wasn't happy about the name either